engine/src/configuration/configuration.cpp

```diff
@@ -42,6 +42,8 @@ Configuration::Configuration() {
 * behavior/range bounding is complicated/necessary-for-stability abd additional validation code is required for checking/adjusting user inputs
 */
 void Configuration::OverrideDefaultsWithUserConfiguration() {
+    general_config.screen = GetGameConfig().GetInt16("general.screen", general_config.screen);
```

Comment:
we should have some kind of method for outputting which monitor is which value.
Perhaps a `vegastrike-engine --list-monitors` which just prints out something like:
```
[0] Primary Display (ASUS...)
[1] Built-in LCD
...
```
where the listing is dynamically generated.
Something like:
```c++
const char * SDL_GetDisplayName(int displayIndex);

for (int i=0; i < SDL_GetNumVideoDisplays(); ++i) {
     const char* name = SDL_GetDisplayName(i);
}
```
only it would have to give a warning about disconnecting/reconnecting displays potentially receiving different values

Looks like like SDL3 is solving some of that but does give that warning with their new `DisplayID` type:
https://wiki.libsdl.org/SDL3/SDL_DisplayID
https://wiki.libsdl.org/SDL3/SDL_GetDisplays

We might want to think through that a little. Should we use the display name (and then compare it to find the right display)? that probably shouldn't change...but it's hard to say.

Reply:
We can, but this is a lot of work for no real reason. For people with 2 monitors it's either 0 or 1. If you're in the wrong monitor, change the number.

Yeah, we could do what you said or even add support to the game to switch monitor in the game menu. But it's overkill and there are more urgent tasks. 

This rebuke is mostly for me. I was halfway through the design in my head while opening issues for _upgrade to SDL 3_ and _this feature_ before I reigned myself in.

Reply:
I'm just thinking through the help desk support side of it and the confusion that may happen and how can we make it a good user experience. At minimum, if we don't do this here, then it should be filed as its own GH Issue with a `help wanted` label for someone to pickup.

Reply:
Yes, outputting which monitors are available would be a nice to have. Maybe just display it in the console output when the game starts?

Reply:
I have tested the changes here, and now it is possible to specify which monitor the game starts up on.